### PR TITLE
Minor tweaks to catalog UI

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/catalog/src/components/CatalogPage/CatalogPage.tsx [hunk 1]
```diff
@@ -137,12 +137,12 @@ const CatalogPage: FC<{}> = () => {
               <span role="img" aria-label="wave" style={{ fontSize: '125%' }}>
                 👋🏼
               </span>
-              Welcome to Backstage, we are happy to have you. Start by checking
-              out our
+              &nbsp;&nbsp;&nbsp;Welcome to Backstage, we are happy to have you.
```
Comment (freben) [inline plugins/catalog/src/components/CatalogPage/CatalogPage.tsx:140]: That sounds like you should have some margin or padding on the waving hand icon's right side instead

File: plugins/catalog/src/components/CatalogPage/CatalogPage.tsx [hunk 2]
```diff
@@ -137,12 +137,12 @@ const CatalogPage: FC<{}> = () => {
...
+              Start by checking out our&nbsp;
               <Link href="/welcome" color="textSecondary">
                 getting started
               </Link>
-              page.
+              &nbsp;page.
```
Comment (freben) [inline plugins/catalog/src/components/CatalogPage/CatalogPage.tsx:145]: Or `{' '}` which has the benefit of not affecting line breaking

**Comment by stefanalund** [comment]
@freben fixed.

File: plugins/catalog/src/components/CatalogPage/CatalogPage.tsx [hunk 3]
```diff
@@ -134,14 +134,18 @@ const CatalogPage: FC<{}> = () => {
           variant="info"
           message={
             <Typography>
-              <span role="img" aria-label="wave" style={{ fontSize: '125%' }}>
+              <span
+                role="img"
+                aria-label="wave"
+                style={{ fontSize: '125%', marginRight: 12 }}
```
Comment (benjdlambert) [inline plugins/catalog/src/components/CatalogPage/CatalogPage.tsx:140]: I'd move this into the `useStyles` block, and do `fontSize: '125%', marginRight: theme.spacing(2)`

Comment (stefanalund) [reply plugins/catalog/src/components/CatalogPage/CatalogPage.tsx]: Yeah, good point. Fixed now.

**Comment by stefanalund** [comment]
@freben @benjdlambert now it works

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**